internal/provider/resource_gitlab_project_runner.go

```diff
@@ -0,0 +1,120 @@
+package provider
+
+import (
+	"log"
+	"strconv"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	gitlab "github.com/xanzy/go-gitlab"
+)
+
+var _ = registerResource("gitlab_project_runner", func() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceGitlabProjectRunnerCreate,
+		Read:   resourceGitlabProjectRunnerRead,
+		Delete: resourceGitlabProjectRunnerDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"project": {
+				Type:     schema.TypeString,
+				ForceNew: true,
+				Required: true,
+			},
+			"runner_id": {
+				Type:     schema.TypeInt,
+				ForceNew: true,
+				Required: true,
+			},
```

Comment:
You'll want to add a `Description` field to each of the entries, so that you can generate documentation correctly. We usually try to adhere to the API documentation description as long as it makes sense, so something like this would work well:

```suggestion
			"project": {
			        Description: "The ID or URL-encoded path of the project owned by the authenticated user",
				Type:     schema.TypeString,
				ForceNew: true,
				Required: true,
			},
			"runner_id": {
			        Description: "The ID of a runner to associate to the project",
				Type:     schema.TypeInt,
				ForceNew: true,
				Required: true,
			},
```

Reply:
I've addressed this in the latest version